pgjdbc/src/main/java/org/postgresql/core/Parser.java

```diff
@@ -1236,11 +1236,12 @@ private static void checkParsePosition(int i, int len, int i0, char[] p_sql,
    * @return the right postgreSql sql
    * @throws SQLException if something goes wrong
    */
-  private static String escapeFunction(String functionName, String args, boolean stdStrings)
+  private static String escapeFunction(String functionName, StringBuilder args, boolean stdStrings)
```

Comment:
should this method be changed to return either `StringBuilder` or `CharSequence`?

Reply:
No It's always a StringBuilder.
It's an internal method

Reply:
It appears there is a catch block which can return a `String` and another place where it returns `StringBuilder.toString()`.
It seems we could change the method signature to return `CharSequence`, which would allow us to directly return a `StringBuilder` rather than converting it to a `String` first.

Reply:
ok my bad I though you wanted to change the input parameter from StringBuilder to CharSequence.
There is room for further improvements but we should create a new issue with all the ideas.

Reply:
Sure. This just seemed to be right in line with this set of proposed changes (and in a method already being changed) to avoid type transformations which do not add value.